packages/askui-nodejs/src/execution/ui-controller-client.ts

```diff
@@ -33,22 +33,22 @@ export class ControlYourUiClient {
 
   ws!: WebSocket;
 
-  connectionState: ClientConnectionState = ClientConnectionState.NOT_CONNECTED;
+  connectionState = UiControllerClientConnectionState.NOT_CONNECTED;
 
   private timeout?: NodeJS.Timeout;
 
-  private currentReject: (reason?: unknown) => void = ControlYourUiClient.EMPTY_REJECT;
+  private currentReject: (reason?: unknown) => void = UiControllerClient.EMPTY_REJECT;
 
   private currentResolve: (
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     value: any,
-  ) => void = ControlYourUiClient.EMPTY_RESOLVE;
+  ) => void = UiControllerClient.EMPTY_RESOLVE;
 
-  constructor(public controlServerUrl: string) { }
+  constructor(public uiControllerUrl: string) { }
```

Comment:
Could also be renamed to `url` as this is clear from context from my perspective